### PR TITLE
fix(Snackbar): fix snackbar positioning and responsiveness

### PR DESCRIPTION
**Pull request by KRRISH96**
### Status :
Ready
##

### Description :
fix snackbar positions and responsiveness

##
### Related Issues :
#287 

## Todos
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by rishichawda — COMMENTED** [review]
Can we have a close button on the indefinite snackbar? Like the one in material ui or similar? I think it would contribute more towards accessibility.

File: lib/snackbar/theme.module.scss [hunk 1]
```diff
@@ -34,10 +34,34 @@
...
 :local(.snackbar) {
-  background: $primary-color;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 42em;
+  background: $cool-blue;
   border-radius: 3px;
-  padding: 10px 20px;
+  padding: 0.7em 1.2em;
```
Comment (kaushiknishchay) [inline lib/snackbar/theme.module.scss]: - make it `padding: 0.9em 1.2em` and it is good to go.

Comment (kaushiknishchay) [inline lib/snackbar/theme.module.scss]: make it padding: 0.9em 1.2em and it is good to go.

**Review by kaushiknishchay — CHANGES_REQUESTED** [review]

File: lib/snackbar/theme.module.scss (hunk 1 above)
Comment (KRRISH96) [reply lib/snackbar/theme.module.scss]: @kaushiknishchay Updated padding in [5994f01](https://github.com/Codebrahma/react-lite-ui/pull/294/commits/5994f01dbd08ff566575a4d7c9f0a3d528e85bd3)

**Review by kaushiknishchay — APPROVED** [review]

**Review by rishichawda — APPROVED** [review]

**Pull request merged**